src/nxdoc/web-ui/nuxeo-elements/quality-assurance.md

```diff
@@ -245,35 +245,38 @@ For more information about how to setup fake responses, please check the officia
-      expect(collections[0].querySelector('.collection-name').textContent).to.be.equal('My Collection'); // and its collection name should be "My Collection"
-    });
+    await waitForEvent(table, 'nuxeo-page-loaded', 1);
+    await flush();
+
+    const collections = table.shadowRoot.querySelectorAll('.collection-box');
+    expect(collections.length).to.be.equal(1); // there should be only one item
+    expect(collections[0].querySelector('.collection-name').textContent).to.be.equal('My Collection'); // and its collection name should be "My Collection
   });
 });
 ```
 
 {{#> callout type='tip' heading='Test Helpers'}}
-Again, the `waitForEvent` method is part of our [test helpers](https://github.com/nuxeo/nuxeo-elements/blob/maintenance-2.4.x/test/test-helpers.js). This method waits for an event to be fired a specific amount of times before returning a promise. Similarly, you can use `waitChanged` to wait for a particular
-property to change on an element, provided that it's set to `notify: true`. Please, check the Polymer documentation on [data-binding](https://polymer-library.polymer-project.org/2.0/docs/devguide/data-binding) for more information on this subject.
+Again, the `waitForEvent` method is part of our [test helpers](https://github.com/nuxeo/nuxeo-elements/blob/maintenance-3.0.x/testing-helpers/index.js). This method waits for an event to be fired a specific amount of times before returning a promise. Similarly, you can use `waitChanged` to wait for a particular
+property to change on an element, provided that it's set to `notify: true`. Please, check the Polymer documentation on [data-binding](https://polymer-library.polymer-project.org/3.0/docs/devguide/data-binding) for more information on this subject.
 {{/callout}}
 
 {{#> callout type='note' heading='Chai API'}}
 For more information about how perform test asserts, please check the official [Chai documentation](http://chaijs.com/api/), which provides both BDD and TDD oriented APIs.
 {{/callout}}
 
-You can then run your tests using `polymer test` or run them interactively via `polymer serve`. Check the Polymer documentation on [WCT](https://polymer-library.polymer-project.org/2.0/docs/tools/tests) for more on this subject.
+You can then run your tests using `polymer test` or run them interactively via `polymer serve`. Check the Polymer documentation on [WCT](https://polymer-library.polymer-project.org/3.0/docs/tools/tests) for more on this subject.
 
 For more examples on testing custom elements, please check our repositories:
-- [nuxeo-elements](https://github.com/nuxeo/nuxeo-elements/tree/maintenance-2.4.x/test)
-- [nuxeo-dataviz-elements](https://github.com/nuxeo/nuxeo-dataviz-elements/tree/maintenance-2.4.x/test)
-- [nuxeo-ui-elements](https://github.com/nuxeo/nuxeo-ui-elements/tree/maintenance-2.4.x/test)
+- [nuxeo-elements](https://github.com/nuxeo/nuxeo-elements/tree/maintenance-3.0.x/testing-helpers)
 
 ### Cucumber
```

Comment:
The cucumber section needs to be review as well. We are still using the syntax to Cucumber v1, and we are already on v4.